bitcoin_client/tests/test_get_master_fingerprint_legacyapp.py

```diff
@@ -0,0 +1,6 @@
+from ledger_bitcoin import Client
+from conftest import SpeculosGlobals
```

Comment:
I need this line to be
```
from .conftest import SpeculosGlobals
```
... to work.

Reply:
I had to make this change because I ran the tests from the parent directory. No need if they are trigger from this `tests/` directory.